### ISSUE TITLE
Is there a trusting trust problem

### ISSUE THREAD

**Issue by kevinburke** [blinsay/homer#2]
Per the example in the README, it looks like you are using the system resolver to resolve the request to Google to get the DNS address you actually want to get.

Is that by design? Should you hardcode an IP address instead?

**Comment by blinsay** [comment]
> Per the example in the README, it looks like you are using the system resolver to resolve the request to Google to get the DNS address you actually want to get.
>
> Is that by design? Should you hardcode an IP address instead?

there's no default resolver at the moment. I'd be happy to put `1.1.1.1/dns-query` in the README if you think that's a better default. I picked `dns.google.com/experimental` just because it was the first resolver on the list in the `curl` wiki.

`homer` uses the system resolver because it was the easiest thing to do. I've got a branch going that lets you specify a resolver to use instead of the system resolver - I could also add an option to disable the system resolver entirely if you think it's useful.
